Content.Shared/StepTrigger/Components/StepTriggerComponent.cs

```diff
@@ -21,6 +21,12 @@ public sealed partial class StepTriggerComponent : Component
     [ViewVariables, AutoNetworkedField]
     public HashSet<EntityUid> CurrentlySteppedOn = new();
 
+    /// <summary>
+    ///     List of entities that left collision this frame.
+    /// </summary>
+    [ViewVariables, AutoNetworkedField]
```

Comment:
datafield so at least this will save

Reply:
I am not sure about this one; two other HashSets with similar usage does not use DataField, but I guess fine, it's easy to add and remove such things